src/main/java/com/srans/nestserver/controller/InvoiceController.java

```diff
@@ -120,4 +135,73 @@ public Invoice createinvoice(@Valid @RequestBody Invoice invoice) {
 		response.put("deleted", Boolean.TRUE);
 		return response;
 	}
+
+	// Invoice generate for multiple tenant...
+	@GetMapping(value = "/invoices/{ids}")
+	@ResponseBody
+	List<Invoice> getTenantById(@PathVariable long[] ids) {
+
+		List<Invoice> invoice = new ArrayList<>();
+
+		for (long id : ids) {
+			Invoice invoiceData = new Invoice();
+
+			System.out.println(id);
+			List<Object> invoiceInfo = invoiceRepository.invoiceData(id);
+			for (Iterator<Object> iterator = invoiceInfo.iterator(); iterator.hasNext();) {
+				Object[] object = (Object[]) iterator.next();
+
+				for (int i = 0; i < object.length; i++) {
+
+					switch (i) {
+					case 0:
```

Comment:
Is there in other way to catch the list , rather than switch?

Reply:
I'll change switch to else if...